code-review/2019_05_10/README.rst

```diff
@@ -381,7 +381,7 @@ However, that does not mean they are not useful.
 
 Whenever any definition is called, it has access to all input arguments that were provided to it (called **locals**) and all variables available in the namespace of the definition (called **globals**).
 Anything higher than the scope of this definition (so, the namespace in which it exists) has no access to the locals, while anything lower sees all locals as globals.
-The following will explain this a bit better, where I use comments to indicate what the globals and locals are in specific scopes (keep in mind that by default, globals can only be viewed and not modified, unless they are declared local using the ``global`` keyword):
+The following will explain this a bit better, where I use comments to indicate what the globals and locals are in specific scopes (keep in mind that by default, globals can only be viewed and not modified, unless they are declared as local using the `global statement <https://docs.python.org/3/reference/simple_stmts.html#global>`_):
```

Comment:
This statement is not unclear - how does one declare "globals" as "local" using the global statement?

Reply:
That is what the link to the documentation is for.

Reply:
But, the just of things is that by using the keyword, you tell Python that you want to use the 'global' variable with that name as a 'local' variable here, instead of creating a new local variable with the same name.